assets/css/shame.scss

```diff
@@ -67,4 +67,18 @@ code {
 
 pre code {
 	word-spacing: normal;
-}
\ No newline at end of file
+}
+
+/* Not sure if this is the right way to implement this functionality. Hugo is not great at dynamic things
+This copies the approach used for 'code-output' or 'View Output' in the docs.
+Displays the currently selected getting-started language notes. This may need to be modified
+once other the kpack and spring boot platforms are added */
+.getting-started__selections {
+	display: none;
+}
+
+.getting-started--selected {
+	.getting-started__selections {
+		display: block;
+	}
+}
```

Comment:
```suggestion
.getting-started__selections.getting-started--selected {
  display: block;
}
```

Reply:
This should unblock the hide/show issue, but I think you have a bug in the JS code.

Reply:
Oh, let me explain the change too. CSS selectors cannot be nested like you have. You need to chain them. The selector I have changed this to basically says the element with both the `getting-started__selections` and `getting-started--selected` classes.